src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -616,7 +617,7 @@ public NeoForgeMod(IEventBus modEventBus, Dist dist, ModContainer container) {
         DualStackUtils.initialise();
         TagConventionLogWarning.init();
 
-        modEventBus.addListener(CapabilityHooks::registerVanillaProviders);
+        modEventBus.addListener(EventPriority.LOW, CapabilityHooks::registerVanillaProviders);
```

Comment:
Only the fallback providers should be registered using `LOW`. (i.e. the ones that might target mod content)

Reply:
Ok I think I got it this time. Thank you for the heads up!